roles/roles.go

```diff
@@ -78,6 +78,17 @@ var (
 	}
 )
 
+var (
+	// services.SSC
```

Comment:
If we declare a new `var` block for every service, than I suggest putting this line of comment before the `var` line. Same for the `services.Dotcom` block.